google/cloud/CMakeLists.txt

```diff
@@ -21,7 +21,7 @@ set(GOOGLE_CLOUD_CPP_IS_RELEASE "")
 
 # This should be included from the top level CMakeLists file
 set(GOOGLE_CLOUD_CPP_VERSION_MAJOR 0)
-set(GOOGLE_CLOUD_CPP_VERSION_MINOR 8)
+set(GOOGLE_CLOUD_CPP_VERSION_MINOR 11)
```

Comment:
s/11/9/?

Reply:
Seeing as how I'm working on the v.0.10.x release of google-cloud-cpp, it seemed appropriate that version should be bumped to 0.11.0. Perhaps I incorrectly inferred that adjustment had been overlooked in prior releases. If the value should have gone from 8->9, I'd like to understand what "component" version that's based on.

Reply:
This is the version number for the "common" components. Not the greatest name for those variable names in hindsight.

Reply:
I'd say you're working on the 0.10.0 release of google-cloud-cpp-bigtable, the 1.1.0 release of google-cloud-cpp-storage, and the 0.8.0 release of google-cloud-cpp(-common).  But that's just my understanding, so I'll defer to someone who knows.

Reply:
changed to 9